ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -205,10 +217,39 @@ Item {
     }
 
     ListView {
```

Comment:
You' need `clip: true` on the ListView if you don't want it to be under the buttons

Reply:
damn this really cleaned up the ui!! thanks!!